documentation/modules/exploit/windows/http/exchange_ecp_dlp_policy.md

```diff
@@ -34,7 +36,7 @@ Follow [Setup](#setup) and [Scenarios](#scenarios).
 
 ### 0
 
-`Exchange Server <= 2016 CU17 and 2019 CU6`
+`Exchange Server <= 2016 CU19 and 2019 CU6`
```

Comment:
Should be [2019 CU8](https://github.com/rapid7/metasploit-framework/pull/14607/files#diff-83a1ea04bd4f5ba53ea3b632b45879ef4f46eaf288f541a2008459bde231f85bR67).

Reply:
Fixed in #14741.